Makefile

```diff
@@ -406,6 +406,14 @@ docker_test:
 		${DEV_DOCKER_IMAGE_NAME}:${DOCKER_IMAGE_TAG} \
 		make test
 
+docker_shell:
```

Comment:
unrelated?

Reply:
My local environment is broken, I need this Makefile entry to debug 😢 Hope it's helpful to others, we already have docker_test to rely on.